web/packages/plugins/sima-application-input/src/EditSimaApplicationInput.tsx

```diff
@@ -60,22 +62,24 @@ const ReadOnlyTextField = (props: { text: string; label: string }) => {
   )
 }
 
-const STaskBlueprint = 'AnalysisPlatformDS/Blueprints/STask'
+const STaskBlueprint =
+  "Ansole.log('tabData', tabDataalysisPlatformDS/Blueprints/STask"
```

Comment:
```suggestion
const STaskBlueprint = "AnalysisPlatformDS/Blueprints/STask"
```
fix typo / accidental paste